### DISCUSSION TITLE
Section navigation

### DISCUSSION THREAD

**Discussion by boxadesign** [ONSdigital/design-system#174] [Feedback]
Use this place to discuss the [section navigation component](https://ons-design-system.netlify.com/components/section-navigation) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Incorrect param name `path` in section navigation component macro-options and example

**Reply by jrbarnes9** [reply]
Fixed in #1612

**Comment by jrbarnes9** [comment]
Allow group headings in section navigation.

Currently, to produce the following lefthand navigation menu, the section navigation component has to be used mutiple times for each sub-headed group. This results in multiple `nav` elements for what is essentially a single navigation menu.

Allowing multiple lists within the component, with sub-headings, will simplify and improve the labelling of the `nav` landmarks on the page.

***Note:*** 
We're already handling these groups/sections in this way in the [sub-navigation macro for the header component](https://github.com/ONSdigital/design-system/blob/a9ae208f024a34b24694c7ddf58994fafb2b69c9/src/components/navigation/_macro.njk#L86)

![Screenshot 2023-02-20 at 12 06 39](https://user-images.githubusercontent.com/43346934/220122994-e569849c-eb89-4625-92d7-f7568700ded4.png)

**Reply by jrbarnes9** [reply]
Raised #2621
